samples/plugins/postgres_jdbc_kerberos/connection-dialog.tcd

```diff
@@ -0,0 +1,12 @@
+<connection-dialog class='postgres_jdbc_kerberos'>
+  <connection-config>
+    <authentication-mode value='ComboBoxIntegrated'/>
+    <authentication-options>
+      <option name="UseWindows" value="auth-integrated" />
```

Comment:
@rosswbrown should we be using `auth-kerberos` in instead of `auth-integrated` ? 
Also, should we be documenting kerberos in the [supported auth modes](https://tableau.github.io/connector-plugin-sdk/docs/auth-modes)

Reply:
@sidwray same question to you about documentation

Reply:
From what I can tell in the code `auth-kerberos` happens to be some teradata specific implementation detail and doesn't have broad platform meaning.   I think this is correct, but I defer to you as person authoring/testing this.  We should probably consider changing this sample over to using modular connection dialog definition, since this is a forward looking sample.  This notion of which auth to use simplifies greatly with that style.

I don't believe we document anything until we're planning to ship this feature, which I thought was post 2020.2.